base/float.jl

```diff
@@ -711,6 +711,74 @@ end
     eps() = eps(Float64)
 end
 
+"""
+    eps(::Type{T}) where T<:AbstractFloat
+    eps()
+
+Returns the *machine epsilon* of the floating point type `T` (`T = Float64` by
+default). This is defined as the gap between 1 and the next largest value representable by
+`T`, and is equivalent to `eps(one(T))`.
+
+```jldoctest
+julia> eps()
+2.220446049250313e-16
+
+julia> eps(Float32)
+1.1920929f-7
+
+julia> 1.0 + eps()
+1.0000000000000002
+
+julia> 1.0 + eps()/2
+1.0
+```
+"""
+eps{T<:AbstractFloat}(::Type{T})
```

Comment:
I think this should also be `eps(::Type{T}) where T<:AbstractFloat`, or `eps(::Type{<:AbstractFloat})`